pkg/system/reconciler.go

```diff
@@ -66,8 +66,11 @@ type Reconciler struct {
 	NooBaa              *nbv1.NooBaa
 	ServiceAccount      *corev1.ServiceAccount
 	CoreApp             *appsv1.StatefulSet
+	NoobaaDB            *appsv1.StatefulSet
```

Comment:
Use capital B for NooBaa -
```suggestion
	NooBaaDB            *appsv1.StatefulSet
```